queries/bash/highlights.scm

```diff
@@ -96,11 +97,13 @@
 (command
   argument: [
              (word) @parameter
-             ((word) @number
-              (#vim-match? @number "^\d*$"))
              (concatenation (word) @parameter)
              ])
 
+(command
+  argument: ((word) @number
+             (#match? @number "^[0-9]+$")))
+
```

Comment:
Can't you "just" match the `word` here? I don't feel the `command` node is useful.

Reply:
There's already a query to match a number as just word, up at line 80.  But if we remove this extra query, then parameters that are also numbers get caught as parameters.

What's the precedence rule?  Last query wins?  It looks as though moving the number query to be after the parameter query works: is it safe to rely on that?  Maybe a bit more fragile to future refactoring?

Reply:
You can safely rely on "last capture wins" as this is the precedence rule we will be going for (for now).